subsys/zigbee/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+zephyr_interface_library_named(zigbee)
+
+target_include_directories(zigbee INTERFACE common)
+target_include_directories(zigbee INTERFACE osif)
+target_include_directories(zigbee INTERFACE zb_error)
```

Comment:
Why is an interface target needed here ?
and not simple using `zephyr_library_include_directories()` on the lib below.